Makefile

```diff
@@ -38,153 +42,172 @@ help:  ## Display this help
-unit-cover-html:
-	go test -coverprofile=cover.out $(GO_TEST_FLAGS) $(PACKAGES)
-	go tool cover -html=cover.out
-
 .PHONY: unit-verbose
 unit-verbose: ## Run unit tests with verbose output
 	VERBOSE=-v make unit
 
+## --------------------------------------
+## Linter Targets
+## --------------------------------------
+
 .PHONY: linters
 linters: sec lint generate-check fmt-check vet ## Run all linters
 
-.PHONY: vet
-vet: ## Run go vet
-	go vet $(PACKAGES)
-
-.PHONY: lint
-lint: golint-binary ## Run golint
-	find $(CODE_DIRS) -type f -name \*.go  |grep -v zz_ | xargs -L1 golint -set_exit_status
-
-.PHONY: generate-check
-generate-check:
-	./hack/generate.sh
-
-.PHONY: generate-check-local
-generate-check-local:
-	IS_CONTAINER=local ./hack/generate.sh
-
 .PHONY: sec
 sec: $(GOPATH)/bin/gosec ## Run gosec
-	gosec -severity medium --confidence medium -quiet $(PACKAGES)
+	gosec -severity medium --confidence medium -quiet ./...
 
 $(GOPATH)/bin/gosec:
 	go get -u github.com/securego/gosec/cmd/gosec
 
+fmt: ## Run go fmt against code
+	go fmt ./...
+
+vet: ## Run go vet against code
+	go vet ./...
+
+.PHONY: lint
+lint: golint-binary ## Run golint
+	find $(CODE_DIRS) -type f -name \*.go  |grep -v zz_ | xargs -L1 golint -set_exit_status
+
 .PHONY: golint-binary
 golint-binary:
 	which golint 2>&1 >/dev/null || $(MAKE) $(GOPATH)/bin/golint
 $(GOPATH)/bin/golint:
 	go get -u golang.org/x/lint/golint
 
-.PHONY: fmt
-fmt: ## Run gofmt and write changes to each file
-	gofmt -l -w $(CODE_DIRS)
-
-.PHONY: fmt-check
-fmt-check: ## Run gofmt and report an error if any changes are made
-	./hack/gofmt.sh
-
-.PHONY: docs
-docs: $(patsubst %.dot,%.png,$(wildcard docs/*.dot))
+## --------------------------------------
+## Build/Run Targets
+## --------------------------------------
 
-%.png: %.dot
-	dot -Tpng $< >$@
+.PHONY:
+build: generate manifests manager tools ## Build everything
 
-.PHONY: e2e-local
-e2e-local:
-	operator-sdk test local ./test/e2e \
-		--namespace $(TEST_NAMESPACE) \
-		--up-local $(SETUP) \
-		$(DEBUG) --go-test-flags "$(GO_TEST_FLAGS)"
+.PHONY: manager
+manager: generate fmt vet ## Build manager binary
+	go build -ldflags $(LDFLAGS) -o bin/manager main.go
 
 .PHONY: run
-run: ## Run the operator outside of a cluster in development mode
-	operator-sdk run --local \
-		--go-ldflags=$(LDFLAGS) \
-		--watch-namespace=$(RUN_NAMESPACE) \
-		--operator-flags="-dev"
+run: generate fmt vet manifests ## Run against the configured Kubernetes cluster in ~/.kube/config
+	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev
 
 .PHONY: demo
-demo: ## Run the operator outside of a cluster using the demo driver
-	operator-sdk run --local \
-		--go-ldflags=$(LDFLAGS) \
-		--watch-namespace=$(RUN_NAMESPACE) \
-		--operator-flags="-dev -demo-mode"
+demo: generate fmt vet manifests ## Run in demo mode
+	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev -demo-mode
+
+install: manifests ## Install CRDs into a cluster
```

Comment:
This should depend on the kustomize target, and use the kustomize variable instead of directly calling kustomize. otherwise you will be using the local install of kustomize, not the hack/tools version of it. This implies different version of kustomize on different machines etc. Same comment for the two targets below

Reply:
Done